packages/react-scripts/scripts/eject.js

```diff
@@ -78,15 +81,15 @@ prompt(
   var appPackage = require(path.join(appPath, 'package.json'));
 
   var ownPackageName = ownPackage.name;
-  console.log('Removing dependency: ' + ownPackageName);
+  console.log('Removing dependency: ' + yellowUnderline(ownPackageName));
```

Comment:
I think this should have the same indentation as “adding a dependency” but they both need to be grouped under “Updating `package.json`“